pkg/usagetracker/usagetrackerpb/usagetracker.proto

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+syntax = "proto3";
+
+package usagetrackerpb;
+
+option go_package = "usagetrackerpb";
+
+import "gogoproto/gogo.proto";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+service UsageTracker {
+  rpc TrackSeries(TrackSeriesRequest) returns (TrackSeriesResponse);
+}
+
+message TrackSeriesRequest {
+  // The tenant owning the series.
+  string userID = 1;
+
+  // The hashes of the series to track.
+  repeated int64 seriesHashes = 2;
```

Comment:
Can this be `uint64`?

Reply:
Done in https://github.com/grafana/mimir/pull/10083/commits/98e96f4fb4a47b03beab5e7ff7111e9cb02e9104